### DISCUSSION TITLE
I can't input Chinese

### DISCUSSION THREAD

**Discussion by miaomiaopu** [Eugeny/tabby#5323] [Q&A] [answered]
After I adjust the tabby font, I can't input Chinese in the Linux server connected with SSH. But it can show Chinese, it's so strange.

**Comment by Myfanqie** [comment]
我也遇到了这样的问题，不能输入中文，可以显示中文

**Reply by miaomiaopu** [reply]
I found a solution. I switched the input method to Microsoft's official Chinese input method

**Comment by miaomiaopu** [comment, marked as answer]
I found a solution. I switched the input method to Microsoft's official Chinese input method

**Reply by lawweiliang** [reply]
Right, I am using this method. It works for me as well.

![image](https://user-images.githubusercontent.com/34936929/228156902-16c86078-f242-4bc3-b75c-5d488fd3e3c4.png)

**Comment by githubZqy** [comment]
除了微软的输入法，别的输入法都不行吗

**Comment by lawweiliang** [comment]
@miaomiaopu  
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
